src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images\star.svg);
```

Comment:
The file path in CSS should use forward slashes, not backslashes. Replace 'images\star.svg' with 'images/star.svg'.